backend/app/processing/kb.py

```
@@ -265,37 +265,54 @@ def search_kb(chatbot_id: str, query: str, k: int = 5) -> List[str]:
         logger.error(f"Error searching knowledge base for {chatbot_id}: {e}")
         return []
 
+from core.config import SUPPORTED_EXTENSIONS
```

Ruff (0.8.2)

Module level import not at top of file

(E402)